README.md

```diff
@@ -16,65 +16,68 @@
+- [Apache Mynewt](https://mynewt.apache.org/)
+- [Apache NuttX](https://nuttx.apache.org/)
+- [RIOT](https://www.riot-os.org/)
+- [Mbed OS](https://os.mbed.com/)
+
+RIOT is currently supported only as a boot target. There is currently no
+defined plan to add any additional port. However, we will accept any new
+port contributed by the community once it is good enough.
 
 ## Using MCUboot
 
-Instructions for different operating systems can be found here:
+Instructions for different operating systems can be found below:
 - [Zephyr](docs/readme-zephyr.md)
-- [Mynewt](docs/readme-mynewt.md)
+- [Apache Mynewt](docs/readme-mynewt.md)
+- [Apache NuttX](docs/readme-nuttx.md)
 - [RIOT](docs/readme-riot.md)
-- [Mbed-OS](docs/readme-mbed.md)
-- [NuttX](docs/readme-nuttx.md)
-- [Espressif IDF](docs/readme-espressif.md)
-- [Simulator](sim/README.rst)
+- [Mbed OS](docs/readme-mbed.md)
+
+There are also instructions for the [Espressif
+IDF](docs/readme-espressif.md) SDK and for the
+[Simulator](sim/README.rst).
 
 ## Roadmap
 
 The issues being planned and worked on are tracked using GitHub issues. To
-participate please visit:
-
-[MCUBoot GitHub Issues](https://github.com/mcu-tools/mcuboot/issues)
-
-~~Issues were previously tracked on [MCUboot JIRA](https://runtimeco.atlassian.net/projects/MCUB/summary)
-, but it is now deprecated.~~
+give your input, visit [MCUBoot GitHub
+Issues](https://github.com/mcu-tools/mcuboot/issues).
 
 ## Browsing
 
-Information and documentation on the bootloader are stored within the source.
-
-~~It was previously also documented on confluence:
-[MCUBoot Confluence](https://runtimeco.atlassian.net/wiki/discover/all-updates)
-however, it is now deprecated and not currently maintained~~
+Information and documentation on the bootloader are stored within the
+source.
 
-For more information in the source, here are some pointers:
+For more information, use the following pointers to the source:
 
 - [boot/bootutil](boot/bootutil): The core of the bootloader itself.
 - [boot/boot\_serial](boot/boot_serial): Support for serial upgrade within the bootloader itself.
-- [boot/zephyr](boot/zephyr): Port of the bootloader to Zephyr
-- [boot/mynewt](boot/mynewt): Mynewt bootloader app
+- [boot/zephyr](boot/zephyr): Port of the bootloader to Zephyr.
+- [boot/mynewt](boot/mynewt): Bootloader application for Mynewt.
 - [boot/nuttx](boot/nuttx): Bootloader application and port of MCUboot interfaces for NuttX.
+- [boot/mbed](boot/mbed): Port of the bootloader to Mbed OS.
 - [boot/espressif](boot/espressif): Bootloader application and MCUboot port for Espressif SoCs.
-- [imgtool](scripts/imgtool.py): A tool to securely sign firmware images for booting by mcuboot.
-- [sim](sim): A bootloader simulator for testing and regression
+- [imgtool](scripts/imgtool.py): A tool to securely sign firmware images for booting by MCUboot.
+- [sim](sim): A bootloader simulator for testing and regression.
 
 ## Joining
 
-Developers welcome!
+Developers are welcome.
```

Comment:
Shouldn't we still use bang at the end of the sentence?

Reply:
We could, but this would be the only sentence in the docs using that. I would say that is not coherent with the style used for the rest of the docs. Still, it is about recruiting people, so, yes, we could keep using that.

Reply:
Ok, I restored the exclamation point.